FILTER_MERGE: 0.00
CALIBRATION_TIME: 0
USE_MAG: 0
GYRO_SAMPLES: 10
GYRO_OFFSET: 14.00000
ACCEL_OFFSET_X: -20
ACCEL_OFFSET_Y: 18
ACCEL_OFFSET_Z: 1989
